src/tests/EdFi.DataManagementService.Tests.E2E/Features/Descriptors/DeleteDescriptorsValidation.feature.disabled

```diff
@@ -0,0 +1,180 @@
+
+Feature: Resources "Delete" Operation validations
+
+        Background:
+            Given the Data Management Service must receive a token issued by "http://localhost"
+              And user is already authorized
+             When a POST request is made to "/ed-fi/absenceEventCategoryDescriptors" with
+                  """
+                    {
+                        "codeValue": "Sick Leave",
+                        "description": "Sick Leave",
+                        "effectiveBeginDate": "2024-05-14",
+                        "effectiveEndDate": "2024-05-14",
+                        "namespace": "uri://ed-fi.org/AbsenceEventCategoryDescriptor",
+                        "shortDescription": "Sick Leave"
+                    }
+                  """
+             Then it should respond with 201 or 200
+
+        @ignore
+        Scenario: 01 Verify deleting a specific descriptor by ID
+             When a DELETE request is made to "/ed-fi/absenceEventCategoryDescriptors/{id}"
+             Then it should respond with 204
+
+        @ignore
+        Scenario: 02 Verify error handling when deleting a descriptor using a invalid id
+             When a DELETE request is made to "/ed-fi/absenceEventCategoryDescriptors/00112233445566"
+             Then it should respond with 404
+              And the response body is
+                  """
+                  {
+                      "detail": "Data validation failed. See 'validationErrors' for details.",
+                      "type": "urn:ed-fi:api:bad-request:data-validation-failed",
+                      "title": "Data Validation Failed",
+                      "status": 404,
+                      "correlationId": null,
+                      "validationErrors": {
+                          "$.id": [
+                              "The value '00112233445566' is not valid."
+                          ]
+                      }
+                  }
+                  """
+
+        @ignore
+        Scenario: 03 Verify error handling when deleting a non existing resource
+            # The id value should be replaced with the resource created in the Background section
+             When a DELETE request is made to "/ed-fi/absenceEventCategoryDescriptors/{id}"
+             Then it should respond with 204
+             When a DELETE request is made to "/ed-fi/absenceEventCategoryDescriptors/{id}"
+             Then it should respond with 404
+
+        @ignore
+        Scenario: 04 Verify response code when GET a deleted resource
+            # The id value should be replaced with the resource created in the Background section
+             When a DELETE request is made to "/ed-fi/absenceEventCategoryDescriptors/{id}"
+             Then it should respond with 204
+             When a GET request is made to "/ed-fi/absenceEventCategoryDescriptors/{id}"
+             Then it should respond with 404
+
+        @ignore
+        Scenario: 05 Verify response when deleting a referenced school
```

Comment:
This scenario is for Resources, not for Descriptors, so it does not belong here.

Reply:
Removed